operator/helm/split_resources.py

```diff
@@ -69,7 +71,11 @@ def helm_release(value: str):
 if __name__ == "__main__":
     exp = args.prefix + "*"
     files = glob.glob(exp)
-    webhookData = '{{- $altNames := list ( printf "seldon-webhook-service.%s" .Release.Namespace ) ( printf "seldon-webhook-service.%s.svc" .Release.Namespace ) -}}\n'
+    webhookData = HELM_CREATERESOURCES_IF_START
+    webhookData = (
+        webhookData
+        + '{{- $altNames := list ( printf "seldon-webhook-service.%s" .Release.Namespace ) ( printf "seldon-webhook-service.%s.svc" .Release.Namespace ) -}}\n'
```

Comment:
Could it not also be seldon-webhook-service.%s.svc.local?

Reply:
this code was copied from example code. So not sure why it would not also contain that. Not seen any issue. I see this as a separate PR if it is an issue.